changedetectionio/html_tools.py

```diff
@@ -67,10 +67,35 @@ def element_removal(selectors: List[str], html_content):
 
 
 # Return str Utf-8 of matched rules
-def xpath_filter(xpath_filter, html_content, append_pretty_line_formatting=False):
+def xpath_filter(xpath_filter, html_content, is_xml=False, append_pretty_line_formatting=False):
     from lxml import etree, html
+    # is_xml is a user flag.
+
+    # This is default case for html_requests.
+    if type(html_content) is bytes and not is_xml:
+        tree = etree.HTML(html_content)
+    # There are special XML cases that need a user intervention(is_xml).
+    elif type(html_content) is bytes and is_xml:
+        tree = etree.XML(html_content)
+    # If type is str.
+    else:
```

Comment:
This part is for webdrivers of Fetcher. Because they don't have bytes(raw_content?)

Reply:
To prove my point, would you provide a way to make a test with the playwright?